pkg/sync/kubernetes/kubernetes_sync.go

```diff
@@ -82,29 +82,33 @@ func (k *Sync) buildConfiguration() (*rest.Config, error) {
 
 func (k *Sync) Notify(ctx context.Context, c chan<- sync.INotify) {
 	if k.ProviderArgs[featureFlagConfigurationName] == "" {
-		k.Logger.Info("No target feature flag configuration set")
+		k.Logger.Error("No target feature flag configuration set")
 		return
 	}
 	if k.ProviderArgs[featureFlagNamespaceName] == "" {
-		k.Logger.Info("No target feature flag configuration namespace set")
+		k.Logger.Error("No target feature flag configuration namespace set")
 		return
 	}
-	k.Logger.Infof("Starting kubernetes sync notifier for resource %s", k.ProviderArgs["featureflagconfiguration"])
+	k.Logger.Info(
```

Comment:
I think we are missing an equivalent log message for updates in the K8s sync.

In filepath, we have:

> `logger.Info("Filepath notifier event: %s %s", event.Name, event.Op.String())`

In HTTP we have:

> `logger.Info("http notifier event: %s has been modified", fs.URI)`

But we have no equivalent for the k8s sync.

Reply:
I have now introduced these logs